### ISSUE TITLE
Setting color without background-color

### ISSUE THREAD

**Issue by strider72** [csstools/sanitize.css#146]
There are places where you set color without also setting background-color. This is poor practice.  Example is a dark gray div with white text, containing a textarea.  In Firefox, sanitize.css changed the textarea's text color to white... on a (default) white background.

This:
```
button,
input,
select,
textarea {
	color: inherit;
	font-size: inherit;
	line-height: inherit;
}
```
...should be this:
```
button,
input,
select,
textarea {
	color: inherit;
	background-color: inherit;
	font-size: inherit;
	line-height: inherit;
}
```

Also:
```
legend {
	box-sizing: border-box; /* 1 */
	color: inherit; /* 2 */
	display: table; /* 1 */
	max-width: 100%; /* 1 */
	padding: 0; /* 3 */
	white-space: normal; /* 1 */
}
```
...should probably be:
```
legend {
	box-sizing: border-box; /* 1 */
	color: inherit; /* 2 */
	background-color: inherit;
	display: table; /* 1 */
	max-width: 100%; /* 1 */
	padding: 0; /* 3 */
	white-space: normal; /* 1 */
}
```
... though I haven't tested that.

**Comment by jonathantneal** [comment]
You are completely right. I will fix this.

**Comment by jonathantneal** [comment]
Fixed in v6.0.0! 🎉
